test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
At this stage we should probably execute `nprocs() > 1 && rmprocs(workers())` which would release resources since the following `node1_tests` would not require them anyways.